CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.4.0 (To be decided, 2019)
+
+* Added get/add/delete functions for vApp template and media item [#225](https://github.com/vmware/go-vcloud-director/pull/225).
```

Comment:
I think this is wrong - did you mean "**metadata** functions"?

Reply:
Fixed